flang/test/Driver/misc-flags.f90

```diff
@@ -10,6 +10,9 @@
 ! Make sure that `-L' is "visible" to Flang's driver
 ! RUN: %flang -L/ -### %s
 
+! Make sure that `-fcommon' is "visible" to Flang's driver
```

Comment:
This is not clear because I didn't leave any comment in this file (mea culpa, sorry), but the other flags tested here are purely driver flags (i.e. there's is nothing for Flang to do here). However, IIUC, `-fcommon` will affect code-generation and that's what we should be testing here instead,

Reply:
So should I put the test for this option somewhere else, or should I still put it in this file?

Reply:
Separate file, please.